packages/cli/test/dev/integration-3.test.ts

```diff
@@ -136,6 +136,33 @@ test(
   })
 );
 
+test('[vercel dev] framework system env vars should be exposed automatically', async () => {
+  const dir = fixture('10a-nextjs-routes');
+  const { dev, port, readyResolver } = await testFixture(dir);
+
+  try {
+    await readyResolver;
+    {
+      const apiRes = await fetch(`http://localhost:${port}/api/env`);
+      validateResponseHeaders(apiRes);
+      const apiJson = await apiRes.json();
+      expect(apiJson.VERCEL_ENV).toBe('development');
+      expect(apiJson.NEXT_PUBLIC_VERCEL_ENV).toBe('development');
+      expect(apiJson.VERCEL_URL).toBe(`localhost:${port}`);
+      expect(apiJson.NEXT_PUBLIC_VERCEL_URL).toBe(`localhost:${port}`);
+
+      const homeRes = await fetch(`http://localhost:${port}/env`);
+      const homeText = await homeRes.text();
+      expect(homeText).toContain('NEXT_PUBLIC_VERCEL_ENV: <!-- -->development');
+      expect(homeText).toContain(
+        `NEXT_PUBLIC_VERCEL_URL: <!-- -->localhost:${port}`
+      );
```

Comment:
Should this test also check for the non-prefixed vars to ensure they are undefined?